ChessGame/static/chessgame/js/ChessBoard/chessBoard.js

```diff
@@ -19,7 +19,10 @@ class ChessBoard {
       console.log(chessBoard);
```

Comment:
Trying to figure out the best way to update chess board - right now creating an update() function that rerenders the chessSquares stored in this.grid